python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -2483,8 +2484,12 @@ def test_bitwise_binops_series(pdf, gdf, binop):
 
 
 @pytest.mark.parametrize("unaryop", [operator.neg, operator.inv, operator.abs])
-def test_unaryops_df(pdf, gdf, unaryop):
-    d = unaryop(pdf - 5)
+@pytest.mark.parametrize("col_name", [None, "abc"])
```

Comment:
Instead, can we not assign `pdf.columns.name` as part of the fixture?

Reply:
Or you can write something like
```python
@pytest.mark.parametrize("col_name,assign_col_name", [(None, False), (None, True), ("abc", True)])
```

and then
```
if assign_col_name:
    pd_df.columns.name = col_name
```

Not a perfect solution but avoids duplicating the entire test.

Reply:
Or use some other sentinel value like `False` or `...` (`Ellipsis`) rather than a boolean flag `assign_col_name`... just trying to be creative. 😄

Reply:
> Or you can write something like
> 
> ```python
> @pytest.mark.parametrize("col_name,assign_col_name", [(None, False), (None, True), ("abc", True)])
> ```
> 
> and then
> 
> ```
> if assign_col_name:
>     pd_df.columns.name = col_name
> ```
> 
> Not a perfect solution but avoids duplicating the entire test.

Used this approach 👍